Source/monster.h

```diff
@@ -157,9 +157,6 @@ struct CMonster {
 		return Anims[static_cast<int>(graphic)];
 	}
 	std::unique_ptr<TSnd> Snds[4][2];
-	uint16_t mMinHP;

```

Comment:
why are these lines being deleted from the struct?

Reply:
The HP variables were only being used in one place and were almost always just a direct copy of the HP data in the `MonstersData` array. After making this change, those variables are no longer used. `mAFNum` is simply unused so I removed it as well.